en/products/vpn/more/vpn-or-proxy.ftl

```diff
@@ -0,0 +1,58 @@
+vpn-or-proxy-isps-can-use = ISPs can use this information for their own ad targeting or for monetization opportunities that could include and sharing your information with third parties interested in data mining, marketing and targeted advertising, which means less privacy and more tracking. Browsing in private mode doesn’t prevent ISPs from seeing where you go online. But sending your web traffic through a web proxy or VPN can make it much harder.
+vpn-or-proxy-when-should-you = When should you choose a VPN or a secure proxy?
+vpn-or-proxy-vpns-and-secure = VPNs and secure web proxies have shared goals: they secure connections. They can, and do, mask your original IP address and protect web traffic that you send between you and your VPN or secure proxy provider. But when would you want to use a VPN vs a proxy?
+vpn-or-proxy-secure-web-proxy = Secure web proxy: browser-level protection
+vpn-or-proxy-a-secure-web = A secure web proxy works for tasks that you might do only in your browser. This can amount to a lot of activity like shopping, paying bills, logging into social media and reading emails. A secure web proxy serves as an intermediary between your browser and the internet. Your web browsing data will pass through a secure tunnel to the internet directly from your browser, masking your IP address, so the web server you are contacting doesn’t know exactly where you are in the world. And that makes you harder to track and target.
+
+# Variables
+#   $url (url) - https://www.consumer.ftc.gov/articles/0014-tips-using-public-wi-fi-networks
+vpn-or-proxy-a-proxy-is-useful = A proxy is useful when you’re browsing the web on a public WiFi. When a proxy is enabled, it will stop eavesdroppers on the same network from spying on your browsing activity or reading your transactions on unencrypted sites. It sounds harmless, but public WiFi networks can be like a backdoor for <a href="{ $url }">hackers</a>.
+
+# Variables
+#   $fpn (url) - https://fpn.firefox.com
+vpn-or-proxy-firefox-private-network = <a href="{ $fpn }">{ -brand-name-firefox-private-network }</a> is an easy to install browser extension that provides a secure, encrypted tunnel to the web to protect your browser connection anywhere you use { -brand-name-firefox }. It’s fast and easy to turn on whenever you need it for extra security in your browser.
+vpn-or-proxy-vpns-devicelevel-protection = VPNs: device-level protection
+vpn-or-proxy-vpns-do-more = VPNs do more than proxies in that a proxy only protects what you do in your browser, whereas a VPN protects all your traffic, including your browser, wherever you have a VPN installed and enabled. VPNs provide added security and privacy for all your online activity—an important consideration if you want to keep your activity to yourself and make it more difficult for data hungry trackers and ISPs to create a profile of you across all your devices, like your phone, computers and tablet.
```

Comment:
- `... like your phone, computers and tablet.` - Lets be consistent  to either use singular or plural forms.
- `... online activity—an important ...` *M dash* with space before or after? All usage in other pages including  this one has extra spaces.